aspnet-core/xml/ns-Microsoft.AspNetCore.Authentication.AzureAD.UI.xml

```diff
@@ -1,6 +1,6 @@
 <Namespace Name="Microsoft.AspNetCore.Authentication.AzureAD.UI">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Contains types that enable better authentication integration with Azure Active Directory.</summary>
+    <remarks>For more information, see <see href="https://docs.microsoft.com/en-us/aspnet/core/security/authentication/azure-active-directory/?view=aspnetcore-3.1">Introduction to authentication with Active Directory in ASP.NET Core</see>.</remarks>
```

Comment:
Update to remove language from the links (so it is auto-detected from the browser) and the version number.
```suggestion
    <remarks>For more information, see <see href="https://docs.microsoft.com/aspnet/core/security/authentication/azure-active-directory/">Introduction to authentication with Active Directory in ASP.NET Core</see>.</remarks>
```